papers/michael_lange/michael_lange.rst

```diff
@@ -402,18 +434,21 @@ object.
                  subs={h: dx, a: 1.})
 
 After building the operator, we can now use it in a time-independent
-convergence loop. However, in this example we need to make sure to
-explicitly exchange the role of the buffers :code:`p` and :code:`pn`.
-This can be achieved by supplying symbolic data objects via keyword
-arguments when invoking the operator, where the name of the argument
-is matched against the name of the original symbol used to create the
-operator. The according initial condition and the resulting
-steady-state solution are depicted in Figures :ref:`fig2dlaplace` and
-:ref:`fig2dlaplacefinal` respectively.
-
-.. raw:: latex
-
-   \pagebreak
+convergence loop that minimizes the :math:`L^1` norm of
+:math:`p`. However, in this example we need to make sure to explicitly
+exchange the role of the buffers :code:`p` and :code:`pn`.  This can
+be achieved by supplying symbolic data objects via keyword arguments
+when invoking the operator, where the name of the argument is matched
+against the name of the original symbol used to create the operator.
+
+The convergence criterion for this example is defined as the relative
+error between two iterations and set to :math:`\Vert p \Vert ^{1} <
+10^{-4}`. The according initial condition and the resulting
+steady-state solution, depicted in Figures :ref:`fig2dlaplace` and
+:ref:`fig2dlaplacefinal` respectively, agree with the original
+tutorial implementation. It should again be noted that the chosen
+numerical scheme might not be optimal to solve steady-state problems
+of this type, since often implicit methods are preferrable.
```

Comment:
since often implicit methods are preferrable -> since implicit methods are often preferred (or fix typo in preferable)